Polyhedron/demo/Polyhedron/MainWindow.h

```diff
@@ -212,6 +212,7 @@ public Q_SLOTS:
 
   /// This slot is used to test exception handling in Qt Scripts.
   void throw_exception();
+
```

Comment:
And not adding that empty line in the .h file will save the need to rebuild all plugin including that header file. You should remove it, so that `MainWindow.h` stays untouched by the PR.